src/test/java/wooteco/subway/acceptance/PathAcceptanceTest.java

```diff
@@ -0,0 +1,101 @@
+package wooteco.subway.acceptance;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import io.restassured.RestAssured;
+import io.restassured.response.ExtractableResponse;
+import io.restassured.response.Response;
+import java.util.List;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpStatus;
+
+public class PathAcceptanceTest extends AcceptanceTest {
+
+    @DisplayName("역이 없는 경우에 404 에러를 발생시킨다.")
+    @Test
+    void searchPathByNotFoundStation() {
+        ExtractableResponse<Response> response = RestAssured.given().log().all()
+            .param("source", 1L)
+            .param("target", 2L)
+            .when()
+            .get("/paths")
+            .then().log().all()
+            .extract();
+
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.NOT_FOUND.value());
+    }
+
+    @DisplayName("source와 target이 같은 경우 400에러를 발생시킨다.")
+    @Test
+    void searchPathBySourceSameAsTarget() {
+        long upStationId = requestCreateStation("강남역").jsonPath().getLong("id");
+        long downStationId = requestCreateStation("역삼역").jsonPath().getLong("id");
+        requestCreateLine("신분당선", "bg-red-600", upStationId, downStationId, 10);
+
+        ExtractableResponse<Response> response = RestAssured.given().log().all()
+            .param("source", upStationId)
+            .param("target", upStationId)
+            .when()
+            .get("/paths")
+            .then().log().all()
+            .extract();
+
+        assertThat(response.statusCode()).isEqualTo(HttpStatus.BAD_REQUEST.value());
+    }
+
+    @DisplayName("지하철 경로 탐색")
+    @Test
+    void searchPath() {
+        long station1 = requestCreateStation("강남역").jsonPath().getLong("id");
+        long station2 = requestCreateStation("역삼역").jsonPath().getLong("id");
+        long station3 = requestCreateStation("잠실역").jsonPath().getLong("id");
+        long station4 = requestCreateStation("선릉역").jsonPath().getLong("id");
```

Comment:
```suggestion
        long 강남역_id = requestCreateStation("강남역").jsonPath().getLong("id");
        long 역삼역_id = requestCreateStation("역삼역").jsonPath().getLong("id");
        long 잠실역_id = requestCreateStation("잠실역").jsonPath().getLong("id");
        long 선릉역_id = requestCreateStation("선릉역").jsonPath().getLong("id");
```

인수 테스트기에 코드를 보는 사람이 이해하기 쉽도록 수정해보았어요. 또한 객체가 아니라 id임을 명시해줬구요. 

읽는 사람에게 명확한 내용을 전달하면 좋겠어요.